source/includes/azure/_instances.md

```diff
@@ -314,4 +314,47 @@ curl -X POST \
 
  <code>POST /services/<a href="#administration-service-connections">:service_code</a>/<a href="#administration-environments">:environment_name</a>/instances/:id?operation=stop</code>
 
- Stop an existing instance. The instance must be in either the power state *PowerState/running* or *PowerState/starting* status for this operation to work. 
+ Stop an existing instance. The instance must be in either the power state *PowerState/running* or *PowerState/starting* status for this operation to work.
+
+
+<!-------------------- ATTACH A DISK -------------------->
+
+#### Attach a disk
+```shell
+curl -X POST \
+   -H "Content-Type: application/json" \
+   -H "MC-Api-Key: your_api_key" \
+   -d "request_body" \
+   "https://cloudmc_endpoint/v1/services/azure-conn/test_env/instances/subscriptions/60a3c9eebe64-55c1-4b1d-969b-60a3c/resourceGroups/azure-connect-system-ssamadh-mean-env/providers/Microsoft.Compute/vm1?operation=attach"
+
+# Request should look like this
+```
+```shell
+# Request Example:
+```
+
+```json
+{
+  "diskToAttach" : "/subscriptions/60a3c9eebe64-55c1-4b1d-969b-60a3c/resourceGroups/azure-connect-system-ssamadh-mean-env/providers/Microsoft.Compute/disks/disktest1",
+  "diskLun": "10"
```

Comment:
It's a bit strange that in 1 case the LUN an integer, and in the other it's a String. Is there a technical reason behind it?

Reply:
Oh my bad i should not have quotes there

Reply:
ab36c1c83d0980c67fd84a0f0cafeaace182f146 fixed